src/pages/Facility/billing/account/AccountSheet.tsx

```diff
@@ -177,7 +177,7 @@ export function AccountSheet({
           <Form {...methods}>
             <form
               onSubmit={methods.handleSubmit(onSubmit)}
-              className="space-y-6 py-6"
+              className="space-y-6 py-6 hide-required-asterisk"
```

Comment:
not right way of adding asterisk

```suggestion
              className="space-y-6 py-6"
```

Reply:
@Janetmaria check line 188 for an example. Ensure aria-required is added to relevant FormLabels.